### DISCUSSION TITLE
How to test style of a component

### DISCUSSION THREAD

**Discussion by ShaktiMandal** [vitest-dev/vitest#2890] [Q&A]
Hi 

I am currently trying to test one of my vue3 component which is structured as given below. Here, I would like to test the style of the second child element (.test-2).  For example, I will check whether background-color is red not. I have tried few options but nothing works for me. 

tried to access - wrapper.find(.test-2) // Getting the element but unable to test the style attributes.

Please let me know if have a solution for this kind of scenarios.

<div class="test">
    <div class="test-1"> </div>
    <div class="test-2"> </div>
</div> 

<style setup> 

.test-2 
   color: white;
  background-color: 'red'
  height: 30px

</style>

**Comment by sheremet-va** [comment]
CSS processing is disabled by default, you need to enable "test.css" option: https://vitest.dev/config/#css
